app/controllers/listings_controller.rb

```diff
@@ -35,7 +41,7 @@ def create
   def update
     if @listing.validate(params[:listing])
       @listing.save
-      redirect_to @listing, notice: 'Listing was successfully updated.'
+      redirect_to listings_path, notice: 'Listing was successfully updated.'
```

Comment:
After updating a Listing (an admin-only function), admin gets taken to listings index